models/obj-model.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
You are going to want to extend your future category and product models from your models.js